include/chimera/mstch.h

```diff
@@ -390,11 +390,26 @@ class Typedef : public ClangWrapper<clang::TypedefNameDecl>
     ::mstch::node classScope() override;
     ::mstch::node scope() override;
     ::mstch::node underlyingClass();
+    ::mstch::node isBuiltinType();
 
 private:
     const clang::CXXRecordDecl *class_decl_;
 };
 
+class BuiltinTypedef : public ClangWrapper<clang::TypedefNameDecl>
+{
+public:
+    BuiltinTypedef(const ::chimera::CompiledConfiguration &config,
+                   const clang::TypedefNameDecl *decl,
+                   const clang::BuiltinType *builtin_type);
+
+    ::mstch::node isBuiltinType();
+    ::mstch::node underlyingBuiltinType();
```

Comment:
Perhaps this can be `underlyingType()`, since we use `type()` as an accessor on other wrappers.

```suggestion
    ::mstch::node underlyingType();
```